Makefile

```diff
@@ -337,13 +337,13 @@ format:
 ###                                  Types                                  ###
 ###############################################################################
 
-EVMOS_URL 		 = https://raw.githubusercontent.com/evmos/ethermint/v0.17.1/crypto/
+EVMOS_URL 		 = https://raw.githubusercontent.com/evmos/ethermint/v0.17.0
 CRYPTO_TYPES 	 = types/crypto
 
 update-deps-types:
 	@mkdir -p $(CRYPTO_TYPES)/ethsecp256k1
-	@curl -sSL $(EVMOS_URL)/ethsecp256k1/keys.pb.go > $(CRYPTO_TYPES)/ethsecp256k1/keys.pb.go
-	@curl -sSL $(EVMOS_URL)/ethsecp256k1/ethsecp256k1.go > $(CRYPTO_TYPES)/ethsecp256k1/ethsecp256k1.go
+	@curl -sSL $(EVMOS_URL)/proto/ethermint/crypto/v1/ethsecp256k1/keys.proto > proto/ethermint/crypto/v1/ethsecp256k1/keys.proto
```

Comment:
Instead of copy pb.go file directly, it is better to download proto file then generate the pb.go file with the specified proto builder which cosmos sdk applies, in order to avoid from the marshaling/unmarshaling issue.